code/ATMOSPHERICS/pipes.dm

```diff
@@ -89,7 +89,9 @@
 		add_fingerprint(user)
 		return 1
 
-	to_chat(user, SPAN_NOTICE("You begin to unfasten \the [src]..."))
+	if(istype(I,/obj/item/weapon/tool/wrench))
```

Comment:
Tools don't have to be specifically wrench, they just need to have QUALITY_BOLT_TURNING flag. 
Plus this doesn't change anything, just put the message under  	if(I.use_tool(user, src, WORKTIME_FAST, QUALITY_BOLT_TURNING, FAILCHANCE_EASY, required_stat = STAT_MEC))